css-brochure-form/styles.css

```diff
@@ -0,0 +1,69 @@
+* {
+  font-family: "Brandon Grotesque", sans-serif;
+  letter-spacing: 0.05em;
+  margin: auto;
+}
+
+h1 {
+  color: rgb(236 92 87);
+  font-size: 1.7rem;
+  text-align: center;
+  font-family: "Brandon Grotesque", sans-serif;
+  padding: 1rem;
+}
+
+label {
+  text-transform: uppercase;
+  font-size: 1.1rem;
+  padding-left: 1rem;
+  font-weight: bold;
+  line-height: 2rem;
+}
+
+form {
+  width: 650px;
+}
+
+div {
+  margin-bottom: 1.5rem;
+}
+
+input {
+  padding: 1rem;
+  font-family: AtlasGrotesk, sans-serif;
+  border-radius: 0.5rem;
+  font-weight: bold;
+  color: rgb(200 200 200);
+}
+
+input,
+select {
+  border-color: rgb(219 219 219);
+  border: solid;
+  border-width: 1.5px;
+  font-size: 1.2rem;
+  width: 650px;
+}
+
+select {
+  background-color: rgb(243 243 243);
+  padding: 1rem 1rem 1rem 0.2rem;
+  border-radius: 0.4rem;
+  border-color: rgb(166 170 169);
+}
+
+.submit-button {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-size: 1.3rem;
+  color: rgb(255 255 255);
+  background-color: rgb(236 92 87);
+  border-radius: 3rem;
+  width: 150px;
+  margin-left: 500px;
```

Comment:
Instead of a really huge left margin, you can either use `float` or wrap your button in a `div` that you style with `text-align: right`